installation/cross_compile/README.md

```diff
@@ -0,0 +1,115 @@
+# Cross-Compilation
+Since processing power of the Raspberry Pi Zero module is insufficient for "quick" compilation we made tools to help you compile ROS2 on your PC for the Raspberry Pi Zero.
+Therefore, this tutorial will give you instructions on how to use the tools to cross-compile ROS2 for Raspberry Pi Zero (could be also extended to other Raspberry Pi versions).
```

Comment:
Is it really specific to the 'zero' version? Since all of them are using the same Raspberry Pi OS, I would assume the same should work on other versions of the Raspberry Pi.

Reply:
The CPU architecture is different. Although, we can scale the to other Raspberry Pi models, but I would like to keep it simple for now.

E.g. Raspberry Pi 3 and Raspberry Pi 4 may only need different optimization flags, but Raspberry Pi Zero and Raspberry Pi 3 need different compilers.